### DISCUSSION TITLE
CVE-2016-20013 do not has title

### DISCUSSION THREAD

**Discussion by testwill** [aquasecurity/trivy#4475] [Q&A] [answered]
### Question

        {
          "VulnerabilityID": "CVE-2016-20013",
          "PkgID": "libc-bin@2.35-0ubuntu3",
          "PkgName": "libc-bin",
          "InstalledVersion": "2.35-0ubuntu3",
          "Layer": {
            "DiffID": "sha256:a790f937a6aea2600982de54a5fb995c681dd74f26968d6b74286e06839e4fb3"
          },
          "SeveritySource": "ubuntu",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2016-20013",
          "DataSource": {
            "ID": "ubuntu",
            "Name": "Ubuntu CVE Tracker",
            "URL": "https://git.launchpad.net/ubuntu-cve-tracker"
          },
          "Description": "sha256crypt and sha512crypt through 0.6 allow attackers to cause a denial of service (CPU consumption) because the algorithm's runtime is proportional to the square of the length of the password.",
          "Severity": "LOW",
          "CweIDs": [
            "CWE-770"
          ],
          "CVSS": {
            "nvd": {
              "V2Vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V2Score": 5,
              "V3Score": 7.5
            }
          },
          "References": [
            "https://akkadia.org/drepper/SHA-crypt.txt",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-20013",
            "https://pthree.org/2018/05/23/do-not-use-sha256crypt-sha512crypt-theyre-dangerous/",
            "https://twitter.com/solardiz/status/795601240151457793"
          ],
          "PublishedDate": "2022-02-19T05:15:00Z",
          "LastModifiedDate": "2022-03-03T16:43:00Z"
        },

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Version

```bash
Version: dev
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-05-26 00:11:21.527698541 +0000 UTC
  NextUpdate: 2023-05-26 06:11:21.527698241 +0000 UTC
  DownloadedAt: 2023-05-26 01:35:16.435786352 +0000 UTC
Java DB:
  Version: 1
  UpdatedAt: 2023-03-12 00:59:02.799086819 +0000 UTC
  NextUpdate: 2023-03-15 00:59:02.799086419 +0000 UTC
  DownloadedAt: 2023-03-12 10:10:22.422625833 +0000 UTC
```

**Comment by knqyf263** [comment, marked as answer]
Yes, Debian doesn't have titles but descriptions. 
https://security-tracker.debian.org/tracker/CVE-2016-20013

Trivy tries to use titles from other distributions, such as Red Hat, but Red Hat doesn't have the information.
https://access.redhat.com/security/cve/cve-2016-20013
